docs/concepts/how-to-grant-api-access.md

```diff
@@ -0,0 +1,183 @@
+
+# Granting Users Access to API Resources in OLM
+
+When operators or cluster extensions are managed via OLM, they often provide Custom Resource Definitions (CRDs) that expose new API resources. Typically, cluster administrators hold full management access to these resources by default, whereas non-administrative users may lack sufficient permissions. Such users often need designated permissions to create, view, or edit these Custom Resources.
```

Comment:
Nit: we might want to use  "cluster extension" instead of operator here and in other place because with OLMv1 we want go beyond operators and be more generic.

This is general comment for the doc: not only for this line.

Reply:
I think by mentioning right off the top that operator or cluster extension are interchangeable it's okay to use operator for brevity in the rest of the doc. Plus it brings along readers who already know OLM as manging operators. Additionally, it's still in the Operator Framework, so it helps still make that connection. Perhaps down the road we start to really root out the all the operator refs?

Reply:
That's fine with me.

cc @michaelryanpeter just in case you have an opinion on the wording.

Reply:
The meeting convey seems to keep ClusterExtension only.